src/test/java/com/example/bookshop/repository/CartItemRepositoryTest.java

```diff
@@ -0,0 +1,63 @@
+package com.example.bookshop.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.example.bookshop.model.CartItem;
+import com.example.bookshop.repository.cartitem.CartItemRepository;
+import java.util.Optional;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.jdbc.Sql;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+public class CartItemRepositoryTest {
```

Comment:
Repository testing is also called integration testing
```suggestion
public class CartItemRepositoryIntegrationTest {
```